docs/DeveloperGuide.adoc

```diff
@@ -844,6 +903,38 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+[discrete]
+=== Use case: See the page of person
+
+*MSS*
+
+1.  User requests to apply the filter of <<social-media-platform,a social media platform>>
```

Comment:
Write it in a A-B-A-B format instead of A-A-B. This is required. It should be a forth-back interaction.